agent/agentbootstrap/bootstrap.go

```diff
@@ -390,9 +391,9 @@ func getEnviron(
 		Config:         modelConfig,
 	}
 	if cloudSpec.Type == cloud.CloudTypeCAAS {
-		return caas.Open(provider, openParams)
+		return caas.Open(stdcontext.TODO(), provider, openParams)
```

Comment:
This should be `stdcontext.Background()`. The TODO one should only be used in tests.